::::: {layout="[0.7, 0.3]"}
::: {#firstcol style="text-align: left;"}
# *Curriculum Vitae* of Eszter Ari
:::

::: {#secondcol style="text-align: right;"}
![](images/Photo.png){width="100px"}
:::
:::::

Bioinformatic Researcher, University Teacher

[arieszter\@gmail.com](mailto:arieszter@gmail.com)

**Websites**

-   Lab website: [Bioinformatics Research Group at Eötvös Loránd University (ELTE), Budapest, H](https://genet.elte.hu/bioinformatics)
-   Personal website at [ELTE, Budapest, H](https://genet.elte.hu/en/content/ari-eszter.t.16148)
-   Personal website at [Biological Research Centre, Szeged, H](http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-member.html#eszter-ari)
-   Public pages: [![Researchgate](images/ResearchGate_icon_SVG.png){width="24"}](https://www.researchgate.net/profile/Eszter-Ari) [![ORCiD](images/ORCID_iD_32x32.svg.png){width="24"}](https://orcid.org/0000-0001-7774-1067) [![ResearcherID](images/web_of_science.png){width="22"}](https://www.webofscience.com/wos/author/record/B-2074-2012) [![Scopus](images/scopus.png){width="24"}](https://www.scopus.com/authid/detail.uri?authorId=26633642900) [![MTMT](images/mtmt.jpg){width="24"}](https://m2.mtmt.hu/api/publication?cond=published;eq;true&cond=core;eq;true&cond=authors.mtid;eq;10020542&sort=publishedYear,desc&sort=firstAuthor,asc&size=20&fields=template,citations:0,citationCount&cite_type=3&&page=1&labelLang=eng) [![X (Twitter)](images/twitter_x_new_logo_square_x_icon.png){width="24"}](https://x.com/EszterAri) [![BlueSky](images/Bluesky_Logo.svg.png){width="27"}](https://bsky.app/profile/eszterari.bsky.social)

# Positions

## Present

**Associate professor - Eötvös Loránd University (ELTE), Department of Genetics**

> H-1117, Pázmány Péter stny. 1/C, Budapest, Hungary

-   2025 - *Role*: Teaching various bioinformatics courses to biology master and graduate students. Supervising undergraduate and graduate students. Coordinating the Molecular Genetics, Cell- and Developmental Biology specialisation.
-   2019 - 2025: assistant professor
-   2009 - 2019: assistant lecturer
-   2007 - 2009: scientific research associate

**Research fellow – HUN-REN**[^cv_en-1] **Biological Research Centre (BRC), Institute of Biochemistry, Szeged**

[^cv_en-1]: Formal names: Research Network of the Hungarian Academy of Science (HAS); Eötvös Loránd Research Network (ELKH)

> H-6726, Temesvári krt. 62, Szeged, Hungary

-   2016 - : Investigating the evolution of antibiotic resistance and virulence using phylogenetic and comparative genomics approaches in Balázs Papp’s Lab, from 2020 as a project leader.

**Research fellow – Hungarian Centre of Excellence for Molecular Medicine (HCEMM), Metabolic Systems Biology Research Group, Szeged**

-   2019 -

## Formerly

**Postdoctoral researcher – Univ. of Veterinary Medicine (Vet-Med Uni), Institut für Populationsgenetik, Vienna, A**

-   2012 - 2014: Analysing the RNA-seq data of thermal adapted fruit fly populations in Christian Schlötterer’s Lab.

# Diplomas & Degrees

-   **Habilitation**: ELTE, Budapest, H, 2023
-   **PhD**: Biology Doctoral School, Theoretical and Evolutionary Biology Doctoral Programme, ELTE, Budapest, H, 2012
-   **Bachelor and Master**: Majoring in Applied Zoology, Faculty of Veterinary Sciences[^cv_en-2], Szent István University, Budapest, H, 1999 - 2004

[^cv_en-2]: Formal and actual name: University of Veterinary Medicine

# Awards & Scholarships

-   Outstanding scientific publication award from Excellence Fund of Eötvös Loránd University, 2024 and 2022, Budapest, H
-   Junior fellowship at Collegium Budapest – Institute for Advanced Study, 2009, Budapest, H
-   Scholarship of the Fac. Veterinary Sci., Szent István Univ., 2003 - 2004, Budapest, H
-   2^nd^ Prize of the Conference for Student Scientists at Fac. Veterinary Sci., Szent István Univ., 2003, Budapest, H

# Grants

-   **Supported Research Groups Programme 2025 - 2028**: *Role*: co-applicant; *Title*: Genomic surveillance for precision therapies against antibiotic-resistant bacteria; *PI*: Papp Balázs; Number: TKCS-2024/66; amount awarded to ELTE: 30,000,000 HUF
-   **ELTE international online courses 2025**: the "Analysis of OMICS Data PR" was selected as a course in [CHARM-EU](https://charm-eu.eu/transnational-course/analysis-of-omics-data-pr/); 500 000 HUF
-   **Hungarian National Research Fund Grant – Postdoctoral Excellence Programme** **2020 - 2023**: *Role*: Principal investigator; *Title*: How do 'superbugs' emerge? - A systematic study of the mobility of resistance and virulence genes in human microbiota and pathogenic bacteria; Nr. [131839](http://nyilvanos.otka-palyazat.hu/index.php?menuid=930&num=131839&lang=EN); 25,500,000 HUF
-   **Promoting Excellence in Education, ELTE 2020**: *Role*: Principal investigator; *Title*: Development of the infrastructure of the computer practical room; 4,000,000 HUF
-   **Visegrad 4 Eastern Partnership (V4Eap) 2014 - 2015**; *Role*: Supervisor; Connected to the fellowship of Zeljko Popovic; *Title*: DORMANCYbase – developing a database on gene and protein expression during dormancy in animals; 3,000 EUR

# Teaching Activities

## Present

-   [Bioinformatics](https://docs.google.com/document/d/1pBIYs0gVawKaT1g_Sf9lYj4QCJCE76MJdwDohb6FXFI/edit?usp=drive_link) (L, P; EN) – for Master students
-   [Analysis of Omics data](https://docs.google.com/document/d/1yh67xqSb7wCUzOfC6KyEP8uik6nOSJF2_dAXLIwa9oI/edit?usp=sharing) (P; EN) – for Master students. The course has also been included in the [CHARM-EU Course Catalogue](https://charm-eu.eu/transnational-course/analysis-of-omics-data-pr/).
-   Bioinformatic seminars (L; EN) – for PhD students
-   [Advanced R programming](https://docs.google.com/document/d/1dELpBDaK-tF1q1sN_qn5w_2mBkjiLxCwtGDT-ZOyKHc/edit?usp=sharing) (P; EN) – for Master students

## Formerly

-   Introduction to Bioinformatics (L; HU) – for Bachelor students
-   Genetic lab. practices (selected P; EN, HU) – for Bachelor students
-   Genomics (selected L; EN, HU) – for Master students
-   Molecular evolution (selected L; HU) – for Master students
-   Evolutionary biology (selected L; HU) – for Master students
-   Discrete mathematical methods in biology (L, P; HU) – for Master and PhD students

## Course Book

-   Genetics Exercises: Chapter 7 – Bioinformatics: Genetic Disease Recognition and Detection (HU)

# Supervising & Mentoring

-   [The personal site at the website of Hungarian Doctoral Council](https://doktori.hu/index.php?menuid=192&lang=EN&sz_ID=11414)
-   **At present**: Co-supervising 3 PhD students, supervising 1 Master, 2 Bachelor, and 3 Scientific Student Association students
-   **Graduated students**: 3 co-supervised PhD, 6 Master, 12 Bachelor, and 6 Scientific Student Association students
-   **Mentoring** **visiting student:** 1 (with Support for summer internships for Hungarian students studying abroad grant)

# Main Workshops

## As Lecturer

-   *ELTE Carpathian Basin Hungarian Summer University*, 2022: Genetic analysis of the virus variants determining the COVID epidemic in Hungary
-   [*Practical Course on Next Generation Sequencing for Population Genetics and Experimental Evolution*](https://www.popgen-vienna.at/training/other-courses/ngs-workshop-2014/), 2014 Vet-Med Uni Vienna, A: Practicals about RNA-seq data analysis

## As Participant

-   *EMBO Training: Laboratory Leadership Course*, 2024, Budapest, H
-   *An R Reproducibility Toolkit for the practical researcher*, 2022, online
-   *8th International Course in Microbial Ecology – Microbiome Metagenome Analysis*, 2017, CNR-ISE, Verbania, I
-   *EMBO course: Bioinformatics and Comparative Genome Analyses*, 2011, Inst. Pasteur Paris, F
-   *Computational Phyloinformatics Course*, 2008, Duke Univ., Durham, NC, USA
-   *Workshop on Molecular Evolution*, 2006, Marine Biological Lab., Woods Hole, MA, USA

# Memberships

-   [Hungarian Society for Bioinformatics](http://mabit.ttk.mta.hu/doc/en/home): 2016 – **member of the directory board**; 2006 – member
-   Hungarian Academy of Sciences, Interdepartmental Scientific Committee on Bioinformatics: 2021 – **committee member**
-   Hungarian node of the [Elixir Europe](https://elixir-europe.org/about-us/who-we-are/nodes/hungary): 2023 – **lead researcher**
-   [Institute of Scientific Computing, Society](https://scicomp.hu/?lang=en) (SciComp): 2024 – member
-   Public Body of the Hungarian Academy of Sciences: 2013 – member (ID: 31344)

# Languages

-   Hungarian – mother tongue
-   English – C1 level
-   Italian – A1 level

# Areas of Interest

-   Bioinformatics
-   Evolutionary Biology
-   Antibiotic Resistance
-   Comparative genomics
-   Molecular phylogenetics
-   Metagenomics
-   Transcriptomics
-   Transcriptional regulation
-   Data analysis

# Professional Skills

-   Developing and using different bioinformatic methods, software and pipelines.
-   *R* programming language – professional level
-   *Linux Shell* scripting – intermediate level

# Editing, Reviewing, & Organizing Conferences

-   [Topic editor in Frontiers in Systems Biology](https://www.frontiersin.org/research-topics/62823/gene-evolution-and-mutation-in-evolutionary-systems-biology)
-   Reviewer in journals: *Brief. Bioinformatics* (3); *Evol. Biol.* (1); *Genome Biol. Evol.* (1); *J. Math. Chem.* (4); *Mol. Ecol.* (1); *Mol. Phylogenetics Evol.* (1); *Sci. Rep.* (1); *Opusc. Zool.* (1); *Plos One* (2)
-   Reviewing: PhD Theses (4), Grant proposals: Hungarian Scientific Research Fund Grant (4)
-   Conference organization as a member of the scientific organizing committee: [The Joint Conference of the Hungarian Physiological Society, the Hungarian Biophysical Society, the Hungarian Society for Microcirculation and Vascular Biology, and the Hungarian Bioinformatics Society](https://www.congress-service.hu/rendezveny/met25), Sep. 2–5, 2025, Szeged, H

# Popularising Science

-   Presenting at the Festival of ELTE: Global map of evolutionary dependencies between antibiotic resistance and virulence genes in *E. coli*. 2022, Budapest, H
-   Creating a website for The genomic epidemiology of SARS-CoV-2 in Hungary
-   Presenting at Capital of Science “Tudományok Fővárosa”: The genomic epidemiology of SARS-CoV-2 in Hungary, 2020, Budapest, H
-   Presenting at Night of Scientists “Kutatók Éjszakája”, 2018 and 2020

# Scientific Websites

-   Supervising the creation of the [TFLink](https://tflink.net/), a transcription factor - target gene database.
-   Supervising the creation of the [mulea](https://cran.r-project.org/web/packages/mulea/) and the [muleaData](https://bioconductor.org/packages/release/data/experiment/html/muleaData.html) R packages for functional enrichment analysis and dataset.

# Publications

```{r pub_tab}
#| echo: false
#| message: false
#| warning: false
options(googlesheets4.timeout = 60)
# read google sheet
pub_tab <- googlesheets4::read_sheet("https://docs.google.com/spreadsheets/d/13qXhW8yNd6KSlE3Taaus-cBUx7O0XkFCkw_IGgVTWNo/edit?usp=sharing")
```

-   [![Google](images/Google_Scholar_logo.svg.png){fig-alt="Google Scholar" width="24"}](https://scholar.google.com/citations?user=cYCKyr0AAAAJ) [![MTMT](images/mtmt.jpg){width="24"}](https://m2.mtmt.hu/api/publication?cond=published;eq;true&cond=core;eq;true&cond=authors.mtid;eq;10020542&sort=publishedYear,desc&sort=firstAuthor,asc&size=20&fields=template,citations:0,citationCount&cite_type=3&&page=1&labelLang=eng)
-   Nr. of published papers: **`r {{ pub_tab$Nr_published_papers[1] }}`** (D1: **`r {{ pub_tab$D1[1] }}`** + Q1: **`r {{ pub_tab$Q1[1] }}`**)
-   Nr. of first, last or corresponding author publications: **`r {{ pub_tab$Nr_firs_last_author[1] }}`**
-   Cumulative impact factor of first, last or corresponding author publications: **`r {{ pub_tab$IF_firs_last_author[1] }}`**
-   Cumulative impact factor of all publications (IF): **`r {{ pub_tab$IF_sum[1] }}`**
-   Independent citations of all publications (IC): **`r {{ pub_tab$IC_sum[1] }}`**
-   h-index: **`r {{ pub_tab$h_index[1] }}`**

## Preprints

```{r pub_list_unpublished}
#| echo: false
#| message: false
#| warning: false
#| results: asis

library(dplyr)

# format the table function
format_publication_unpublished <- function(authors, year, title, journal, 
                                           volume, note) {
  # Add note only if it is not NA
  note_part <- if(!is.na(note)) paste0(" ", note) else ""
  paste0("- ", authors, " (", year, ") ", title, " *", journal, "*, ", volume,
    ".", note_part)}

# Generate formatted publication list
pub_tab_unpublished <- pub_tab %>% 
  # unpublished
  filter(published == "no")

pub_list_unpublished <- apply(pub_tab_unpublished, 1, function(row) {
  format_publication_unpublished(row["authors"], row["year"], row["title"], 
                                 row["journal"], row["volume"], row["note"])})

# Print the publication list
cat(paste(pub_list_unpublished, collapse = "\n\n"))
```

## Published

```{r pub_list_published}
#| echo: false
#| message: false
#| warning: false
#| results: asis

# Format the table function
format_publication_published <- function(authors, year, title, journal, volume, 
                                         Q, IF, IC, note, important) {
  # Add note only if it is not NA
  note_part <- if (!is.na(note)) paste0(" ", note) else ""
  # Apply green color styling if important is "yes"
  text <- paste0(authors, " (", year, ") ", title, " *", journal, "*, ", volume,
                 ". **", Q, ", IF: ", IF, ", IC: ", IC, note_part, "**")
  # Check for "important" being "yes" while handling NA
  if (!is.na(important) && important == "yes") {
    text <- paste0("- ", "<span style='color: #00ab44;'>", text, "</span>")
  } else {
    text <- paste0("- ", text)
  }
  return(text)
}

# Generate formatted publication list
pub_tab_published <- pub_tab %>% 
  # Filter for published
  filter(published == "yes")

pub_list_published <- apply(pub_tab_published, 1, function(row) {
  format_publication_published(row["authors"], row["year"], row["title"], 
                               row["journal"], row["volume"], row["Q"], 
                               row["IF"], row["IC"], row["note"], 
                               row["important"])})

# Print the publication list
cat(paste(pub_list_published, collapse = "\n\n"))
```

# Oral Presentations in Conferences

-   Nr. of international conference talks: **9**
-   Nr. of national conference talks: **6**
-   As an invited speaker\*: **3**

<!-- Insert a blank line here -->

-   Ari E, *et al.* (2024) mulea - an R package for enrichment analysis using multiple ontologies and empirical false discovery rate. [*Bioinformatika, a Magyar Tudomány Ünnepén*](https://www.mabit.org.hu/doc/hu/society/conferences/2024/bioinf_2024/1), 8 Nov, Budapest, H
-   Ari E, *et al.* (2024) mulea - an R package for enrichment analysis using multiple ontologies and empirical false discovery rate. [*SciComp24 Conference*](https://indico.wigner.hu/event/1583/overview), 17-19 Oct, Szeged, H
-   Ari E, *et al.* (2023) Global map of evolutionary dependencies between antibiotic resistance and virulence genes in *E. coli*. [*3rd HCEMM PhD-POSTDOC Symposium*](https://www.hcemm.eu/news/2023/11/3rd-hpps-phd-postdoc-symposium-keszthely/), 8-9 Nov, Keszthely, H
-   Ari E, *et al.* (2022) Global map of evolutionary dependencies between antibiotic resistance and virulence genes in *E. coli*. [*1st Bioinformatics and Data Science in Genomic Studies (BDG2022)*](https://konferencia.unideb.hu/en/BDG2022) online conference, 25 Nov, University of Debrecen, H
-   Ari E, *et al.* (2022) Global map of evolutionary dependencies between antibiotic resistance and virulence genes in *E. coli*. *Bioinformatics, Celebrating the Hungarian Science*, 11 Nov, HAS, Research Centre for Natural Sciences, Budapest, H
-   Ari E\*, Kintses B (2020) Methods to investigate the microbiome and understand the results. *Meeting of Hungarian Society for Gastroenterology, Section Colon*, 6-7 Mar, Visegrád, H
-   Ari E, *et al.* (2019) Phylogenetic barriers to horizontal transfer of antimicrobial peptide resistance genes in the human gut microbiota. *EvolBiol Day*, 17 Apr, Szeged, H
-   Ari E\*, *et al.* (2018) How well do the antimicrobial peptide resistance genes spread? *Bioinformatics, Celebrating the Hungarian Science*, 16 Nov, HAS, Research Centre for Natural Sciences, Budapest, H
-   Ari E: (2017) MulEA – A tool for multi-enrichment analysis. *2nd Interdisciplinary Signaling Workshop*, 17-21 Jul, Visegrád, H
-   Ari E\*: (2012) Molecular phylogenetic reconstructions with a discrete mathematical method, the Boolean analysis. *International Congress, Hungarian Society for Microbiology*, 24-26 Oct, Keszthely, H
-   Ari E & Jakó É (2012) Comparison of Boolean analysis and standard phylogenetic methods using artificially evolved and natural mt-tRNA sequences from great apes. *9th Joint Conference on Mathematics and Computer Science*, 9-12 Feb, Siófok, H
-   Ari E, *et al.* (2007) Boolean analysis: A new discrete mathematical method for phylogenetic reconstruction. *From Molecular Informatics to Bioinformatics – International Symposium*, 19-21 Apr, Collegium Budapest, H
-   Ari E & Jakó É (2007) Testing a new discrete mathematical method for phylogenetic reconstruction. *Evolution 2007 Conference*, 16-20 Jun, Christchurch, NZ
-   Ari E & Jakó É (2007) Testing a new discrete mathematical method for phylogenetic reconstruction. *The Dumont D'Urville Workshop on Applied Evolutionary Bioinformatics*, 24-27 Jun, Kaikoura, NZ
-   Ari E, *et al.* (2007) Phylogenetic tree reconstruction with a new discrete mathematical method. *Molecular taxonomic, phylogenetic and phylogeographic researches in Hungary, Meeting in memory of Sámuel Diószegi*, 17 Nov, Debrecen, H
-   Ari E (2006) Reconstructing the phylogenetic tree of great apes by using a new discrete mathematical method. *12th Annual European Meeting of PhD students in Evolutionary Biology*, 4-9 Sep, St. Andrews, UK

# Other Invited Talks

-   Ari E\* (2017) Investigating the antimicrobial peptide resistome in the human gut microbiome: a metagenomic approach. University of Graz, Institute of Zoology, 15 Nov, Graz, A
-   Ari E\*: (2015) Rapid evolution of phenotypic plasticity during experimental evolution of *Drosophila*. The Genome Analysis Centre, 23 Feb, Norwich, UK

# Poster Presentations in Conferences

-   Nr. of international conference posters: **9**

<!-- Insert a blank line here -->

-   Ari E, *et al.* (2023) [Global map of evolutionary dependencies between antibiotic resistance and virulence genes in *E. coli*](https://indico.ictp.it/event/10210/material/4/0.pdf), [*EMBO Workshop: Plasmids as vehicles of AMR spread*](https://meetings.embo.org/event/23-amr-plasmids), 12-18 Sep, (Trieste, I) online
-   Ari E, *et al.* (2023) Global map of evolutionary dependencies between antibiotic resistance and virulence genes in *E. coli*, [*EMBO Workshop: Predicting evolution*](https://www.embl.org/about/info/course-and-conference-office/events/pev23-01/), 11-14 Jul, Heidelberg, D
-   Ari E, *et al.* (2022) [Global map of evolutionary dependencies between antibiotic resistance and virulence genes in *E. coli*](https://www.lamg.info/_files/ugd/2e355c_d62492daad5840479e27865c3233bfba.pdf). [*Lake Arrowhead Microbial Genomics Conference*](https://www.lamg.info/), 11-15 Sep, Lake Arrowhead, CA, USA
-   Rutka M, Szántó K, Bacsur P, Resál T, Jójárt B, Bálint A, **Ari E**, Kintses B, Fehér T, Asbóth A, Pigniczki D, Bor R, Fábián A, Farkas K, Maléth J, Szepes Z & Molnár T (2022) [P713 Gut microbiota alterations after bowel preparation amongst inflammatory bowel disease patients](https://academic.oup.com/ecco-jcc/article/16/Supplement_1/i609/6513059). *Journal of Crohn’s and Colitis*, 16 (Supplement_1): i609. **Q1, IF: 9.07**
-   Ari E, *et al.* (2019) [Phylogenetic barriers to horizontal transfer of antimicrobial peptide resistance genes in the human gut microbiota](https://www.researchgate.net/publication/338231106_Phylogenetic_barriers_to_horizontal_transfer_of_AMP_resistance_genes). *Gordon Research Conference on Molecular Mechanisms in Evolution*, 9-14 Jun, Easton, MA, USA
-   Ari E, *et al.* (2019) [Phylogenetic barriers to horizontal transfer of antimicrobial peptide resistance genes in the human gut microbiota](https://www.researchgate.net/publication/338231106_Phylogenetic_barriers_to_horizontal_transfer_of_AMP_resistance_genes). *Gordon Research Seminar on Molecular Mechanisms in Evolution*, 8-9 Jun, Easton, MA, USA
-   Ari E & Jakó É (2015) Graph-based generalized Boolean descriptors for classification of biological macromolecules. *Conferentia Chemometrica*, 13-16 Sep, Budapest, H
-   Ari E *et al.* (2015) Rapid evolution of phenotypic plasticity during experimental evolution of Drosophila. *Annual Meeting of the Society for Molecular Biology and Evolution*, 12-16 Jul, Vienna, A
-   Ari E, *et al.* (2006) Testing a new discrete mathematical method for reconstructing the great apes phylogeny based on mitochondrial tRNAs. *Evolution of Biomolecular Structure*, 25-27 May, Vienna, A
