<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.40">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>cv_en</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-549806ee2085284f45b00abea8c6df48.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-dark-8ea72dc5fed832574809a9c94082fbbb.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-45f760928dae0b147bf9b9b73d39efb8.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="site_libs/bootstrap/bootstrap-dark-40f906436e47dee42241104adbff32a0.min.css" rel="prefetch" append-hash="true" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


<link rel="stylesheet" href="styles.css">
<link rel="canonical" href="https://barizona.github.io/cv/">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html"> 
<span class="menu-text">üá¨üáß Curriculum Vitae in English</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./cv_hu.html"> 
<span class="menu-text">üá≠üá∫ Curriculum Vitae in Hunagrian</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul class="collapse">
  <li><a href="#positions" id="toc-positions" class="nav-link active" data-scroll-target="#positions">Positions</a>
  <ul class="collapse">
  <li><a href="#present" id="toc-present" class="nav-link" data-scroll-target="#present">Present</a></li>
  <li><a href="#formerly" id="toc-formerly" class="nav-link" data-scroll-target="#formerly">Formerly</a></li>
  </ul></li>
  <li><a href="#diplomas-degrees" id="toc-diplomas-degrees" class="nav-link" data-scroll-target="#diplomas-degrees">Diplomas &amp; Degrees</a></li>
  <li><a href="#awards-scholarships" id="toc-awards-scholarships" class="nav-link" data-scroll-target="#awards-scholarships">Awards &amp; Scholarships</a></li>
  <li><a href="#grants" id="toc-grants" class="nav-link" data-scroll-target="#grants">Grants</a></li>
  <li><a href="#teaching-activities" id="toc-teaching-activities" class="nav-link" data-scroll-target="#teaching-activities">Teaching Activities</a>
  <ul class="collapse">
  <li><a href="#present-1" id="toc-present-1" class="nav-link" data-scroll-target="#present-1">Present</a></li>
  <li><a href="#formerly-1" id="toc-formerly-1" class="nav-link" data-scroll-target="#formerly-1">Formerly</a></li>
  <li><a href="#course-book" id="toc-course-book" class="nav-link" data-scroll-target="#course-book">Course Book</a></li>
  </ul></li>
  <li><a href="#supervising-mentoring" id="toc-supervising-mentoring" class="nav-link" data-scroll-target="#supervising-mentoring">Supervising &amp; Mentoring</a></li>
  <li><a href="#main-workshops" id="toc-main-workshops" class="nav-link" data-scroll-target="#main-workshops">Main Workshops</a>
  <ul class="collapse">
  <li><a href="#as-lecturer" id="toc-as-lecturer" class="nav-link" data-scroll-target="#as-lecturer">As Lecturer</a></li>
  <li><a href="#as-participant" id="toc-as-participant" class="nav-link" data-scroll-target="#as-participant">As Participant</a></li>
  </ul></li>
  <li><a href="#memberships" id="toc-memberships" class="nav-link" data-scroll-target="#memberships">Memberships</a></li>
  <li><a href="#languages" id="toc-languages" class="nav-link" data-scroll-target="#languages">Languages</a></li>
  <li><a href="#areas-of-interest" id="toc-areas-of-interest" class="nav-link" data-scroll-target="#areas-of-interest">Areas of Interest</a></li>
  <li><a href="#professional-skills" id="toc-professional-skills" class="nav-link" data-scroll-target="#professional-skills">Professional Skills</a></li>
  <li><a href="#editing-reviewing-organizing-conferences" id="toc-editing-reviewing-organizing-conferences" class="nav-link" data-scroll-target="#editing-reviewing-organizing-conferences">Editing, Reviewing, &amp; Organizing Conferences</a></li>
  <li><a href="#popularising-science" id="toc-popularising-science" class="nav-link" data-scroll-target="#popularising-science">Popularising Science</a></li>
  <li><a href="#scientific-websites" id="toc-scientific-websites" class="nav-link" data-scroll-target="#scientific-websites">Scientific Websites</a></li>
  <li><a href="#publications" id="toc-publications" class="nav-link" data-scroll-target="#publications">Publications</a>
  <ul class="collapse">
  <li><a href="#preprints" id="toc-preprints" class="nav-link" data-scroll-target="#preprints">Preprints</a></li>
  <li><a href="#published" id="toc-published" class="nav-link" data-scroll-target="#published">Published</a></li>
  </ul></li>
  <li><a href="#oral-presentations-in-conferences" id="toc-oral-presentations-in-conferences" class="nav-link" data-scroll-target="#oral-presentations-in-conferences">Oral Presentations in Conferences</a></li>
  <li><a href="#other-invited-talks" id="toc-other-invited-talks" class="nav-link" data-scroll-target="#other-invited-talks">Other Invited Talks</a></li>
  <li><a href="#poster-presentations-in-conferences" id="toc-poster-presentations-in-conferences" class="nav-link" data-scroll-target="#poster-presentations-in-conferences">Poster Presentations in Conferences</a></li>
  </ul>
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="cv_en.pdf"><i class="bi bi-file-pdf"></i>PDF</a></li><li><a href="cv_en.docx"><i class="bi bi-file-word"></i>MS Word</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content"><header id="title-block-header" class="quarto-title-block"><h1 class="title display-7"><em>Curriculum Vitae</em> of Eszter Ari</h1></header>




<div>

</div>
<div class="quarto-layout-panel" data-layout="[0.7, 0.3]">
<div class="quarto-layout-row">
<div id="firstcol" class="quarto-layout-cell" style="flex-basis: 70.0%;justify-content: flex-start;">
<section id="curriculum-vitae-of-eszter-ari" class="level1">

</section>
</div>
<div id="secondcol" class="quarto-layout-cell" style="flex-basis: 30.0%;justify-content: center;">
<p><img src="images/Photo.png" class="img-fluid" width="100"></p>
</div>
</div>
</div>
<p>Bioinformatic Researcher, University Teacher</p>
<p><a href="mailto:arieszter@gmail.com">arieszter@gmail.com</a></p>
<p><strong>Websites</strong></p>
<ul>
<li>Lab website: <a href="https://genet.elte.hu/bioinformatics">Bioinformatics Research Group at E√∂tv√∂s Lor√°nd University (ELTE), Budapest, H</a></li>
<li>Personal website at <a href="https://genet.elte.hu/en/content/ari-eszter.t.16148">ELTE, Budapest, H</a></li>
<li>Personal website at <a href="http://group.szbk.u-szeged.hu/sysbiol/papp-balazs-lab-member.html#eszter-ari">Biological Research Centre, Szeged, H</a></li>
<li>Public pages: <a href="https://www.researchgate.net/profile/Eszter-Ari"><img src="images/ResearchGate_icon_SVG.png" class="img-fluid" width="24" alt="Researchgate"></a> <a href="https://orcid.org/0000-0001-7774-1067"><img src="images/ORCID_iD_32x32.svg.png" class="img-fluid" width="24" alt="ORCiD"></a> <a href="https://www.webofscience.com/wos/author/record/B-2074-2012"><img src="images/web_of_science.png" class="img-fluid" width="22" alt="ResearcherID"></a> <a href="https://www.scopus.com/authid/detail.uri?authorId=26633642900"><img src="images/scopus.png" class="img-fluid" width="24" alt="Scopus"></a> <a href="https://m2.mtmt.hu/api/publication?cond=published;eq;true&amp;cond=core;eq;true&amp;cond=authors.mtid;eq;10020542&amp;sort=publishedYear,desc&amp;sort=firstAuthor,asc&amp;size=20&amp;fields=template,citations:0,citationCount&amp;cite_type=3&amp;&amp;page=1&amp;labelLang=eng"><img src="images/mtmt.jpg" class="img-fluid" width="24" alt="MTMT"></a> <a href="https://x.com/EszterAri"><img src="images/twitter_x_new_logo_square_x_icon.png" class="img-fluid" width="24" alt="X (Twitter)"></a> <a href="https://bsky.app/profile/eszterari.bsky.social"><img src="images/Bluesky_Logo.svg.png" class="img-fluid" width="27" alt="BlueSky"></a></li>
</ul>
<section id="positions" class="level1">
<h1>Positions</h1>
<section id="present" class="level2">
<h2 class="anchored" data-anchor-id="present">Present</h2>
<p><strong>Associate professor - E√∂tv√∂s Lor√°nd University (ELTE), Department of Genetics</strong></p>
<blockquote class="blockquote">
<p>H-1117, P√°zm√°ny P√©ter stny. 1/C, Budapest, Hungary</p>
</blockquote>
<ul>
<li>2025 - <em>Role</em>: Teaching various bioinformatics courses to biology master and graduate students. Supervising undergraduate and graduate students. Coordinating the Molecular Genetics, Cell- and Developmental Biology specialisation.</li>
<li>2019 - 2025: assistant professor</li>
<li>2009 - 2019: assistant lecturer</li>
<li>2007 - 2009: scientific research associate</li>
</ul>
<p><strong>Research fellow ‚Äì HUN-REN</strong><a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a> <strong>Biological Research Centre (BRC), Institute of Biochemistry, Szeged</strong></p>
<blockquote class="blockquote">
<p>H-6726, Temesv√°ri krt. 62, Szeged, Hungary</p>
</blockquote>
<ul>
<li>2016 - : Investigating the evolution of antibiotic resistance and virulence using phylogenetic and comparative genomics approaches in Bal√°zs Papp‚Äôs Lab, from 2020 as a project leader.</li>
</ul>
<p><strong>Research fellow ‚Äì Hungarian Centre of Excellence for Molecular Medicine (HCEMM), Metabolic Systems Biology Research Group, Szeged</strong></p>
<ul>
<li>2019 -</li>
</ul>
</section>
<section id="formerly" class="level2">
<h2 class="anchored" data-anchor-id="formerly">Formerly</h2>
<p><strong>Postdoctoral researcher ‚Äì Univ. of Veterinary Medicine (Vet-Med Uni), Institut f√ºr Populationsgenetik, Vienna, A</strong></p>
<ul>
<li>2012 - 2014: Analysing the RNA-seq data of thermal adapted fruit fly populations in Christian Schl√∂tterer‚Äôs Lab.</li>
</ul>
</section>
</section>
<section id="diplomas-degrees" class="level1">
<h1>Diplomas &amp; Degrees</h1>
<ul>
<li><strong>Habilitation</strong>: ELTE, Budapest, H, 2023</li>
<li><strong>PhD</strong>: Biology Doctoral School, Theoretical and Evolutionary Biology Doctoral Programme, ELTE, Budapest, H, 2012</li>
<li><strong>Bachelor and Master</strong>: Majoring in Applied Zoology, Faculty of Veterinary Sciences<a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a>, Szent Istv√°n University, Budapest, H, 1999 - 2004</li>
</ul>
</section>
<section id="awards-scholarships" class="level1">
<h1>Awards &amp; Scholarships</h1>
<ul>
<li>Outstanding scientific publication award from Excellence Fund of E√∂tv√∂s Lor√°nd University, 2024 and 2022, Budapest, H</li>
<li>Junior fellowship at Collegium Budapest ‚Äì Institute for Advanced Study, 2009, Budapest, H</li>
<li>Scholarship of the Fac. Veterinary Sci., Szent Istv√°n Univ., 2003 - 2004, Budapest, H</li>
<li>2<sup>nd</sup> Prize of the Conference for Student Scientists at Fac. Veterinary Sci., Szent Istv√°n Univ., 2003, Budapest, H</li>
</ul>
</section>
<section id="grants" class="level1">
<h1>Grants</h1>
<ul>
<li><strong>Supported Research Groups Programme 2025 - 2028</strong>: <em>Role</em>: co-applicant; <em>Title</em>: Genomic surveillance for precision therapies against antibiotic-resistant bacteria; <em>PI</em>: Papp Bal√°zs; Number: TKCS-2024/66; amount awarded to ELTE: 30,000,000 HUF</li>
<li><strong>ELTE international online courses 2025</strong>: the ‚ÄúAnalysis of OMICS Data PR‚Äù was selected as a course in <a href="https://charm-eu.eu/transnational-course/analysis-of-omics-data-pr/">CHARM-EU</a>; 500 000 HUF</li>
<li><strong>Hungarian National Research Fund Grant ‚Äì Postdoctoral Excellence Programme</strong> <strong>2020 - 2023</strong>: <em>Role</em>: Principal investigator; <em>Title</em>: How do ‚Äòsuperbugs‚Äô emerge? - A systematic study of the mobility of resistance and virulence genes in human microbiota and pathogenic bacteria; Nr. <a href="http://nyilvanos.otka-palyazat.hu/index.php?menuid=930&amp;num=131839&amp;lang=EN">131839</a>; 25,500,000 HUF</li>
<li><strong>Promoting Excellence in Education, ELTE 2020</strong>: <em>Role</em>: Principal investigator; <em>Title</em>: Development of the infrastructure of the computer practical room; 4,000,000 HUF</li>
<li><strong>Visegrad 4 Eastern Partnership (V4Eap) 2014 - 2015</strong>; <em>Role</em>: Supervisor; Connected to the fellowship of Zeljko Popovic; <em>Title</em>: DORMANCYbase ‚Äì developing a database on gene and protein expression during dormancy in animals; 3,000 EUR</li>
</ul>
</section>
<section id="teaching-activities" class="level1">
<h1>Teaching Activities</h1>
<section id="present-1" class="level2">
<h2 class="anchored" data-anchor-id="present-1">Present</h2>
<ul>
<li><a href="https://docs.google.com/document/d/1pBIYs0gVawKaT1g_Sf9lYj4QCJCE76MJdwDohb6FXFI/edit?usp=drive_link">Bioinformatics</a> (L, P; EN) ‚Äì for Master students</li>
<li><a href="https://docs.google.com/document/d/1yh67xqSb7wCUzOfC6KyEP8uik6nOSJF2_dAXLIwa9oI/edit?usp=sharing">Analysis of Omics data</a> (P; EN) ‚Äì for Master students. The course has also been included in the <a href="https://charm-eu.eu/transnational-course/analysis-of-omics-data-pr/">CHARM-EU Course Catalogue</a>.</li>
<li>Bioinformatic seminars (L; EN) ‚Äì for PhD students</li>
<li><a href="https://docs.google.com/document/d/1dELpBDaK-tF1q1sN_qn5w_2mBkjiLxCwtGDT-ZOyKHc/edit?usp=sharing">Advanced R programming</a> (P; EN) ‚Äì for Master students</li>
</ul>
</section>
<section id="formerly-1" class="level2">
<h2 class="anchored" data-anchor-id="formerly-1">Formerly</h2>
<ul>
<li>Introduction to Bioinformatics (L; HU) ‚Äì for Bachelor students</li>
<li>Genetic lab. practices (selected P; EN, HU) ‚Äì for Bachelor students</li>
<li>Genomics (selected L; EN, HU) ‚Äì for Master students</li>
<li>Molecular evolution (selected L; HU) ‚Äì for Master students</li>
<li>Evolutionary biology (selected L; HU) ‚Äì for Master students</li>
<li>Discrete mathematical methods in biology (L, P; HU) ‚Äì for Master and PhD students</li>
</ul>
</section>
<section id="course-book" class="level2">
<h2 class="anchored" data-anchor-id="course-book">Course Book</h2>
<ul>
<li>Genetics Exercises: Chapter 7 ‚Äì Bioinformatics: Genetic Disease Recognition and Detection (HU)</li>
</ul>
</section>
</section>
<section id="supervising-mentoring" class="level1">
<h1>Supervising &amp; Mentoring</h1>
<ul>
<li><a href="https://doktori.hu/index.php?menuid=192&amp;lang=EN&amp;sz_ID=11414">The personal site at the website of Hungarian Doctoral Council</a></li>
<li><strong>At present</strong>: Co-supervising 3 PhD students, supervising 2 Master, 2 Bachelor, and 3 Scientific Student Association students</li>
<li><strong>Graduated students</strong>: 3 co-supervised PhD, 6 Master, 12 Bachelor, and 6 Scientific Student Association students</li>
<li><strong>Mentoring</strong> <strong>visiting student:</strong> 1 (with Support for summer internships for Hungarian students studying abroad grant)</li>
</ul>
</section>
<section id="main-workshops" class="level1">
<h1>Main Workshops</h1>
<section id="as-lecturer" class="level2">
<h2 class="anchored" data-anchor-id="as-lecturer">As Lecturer</h2>
<ul>
<li><em>ELTE Carpathian Basin Hungarian Summer University</em>, 2022: Genetic analysis of the virus variants determining the COVID epidemic in Hungary</li>
<li><a href="https://www.popgen-vienna.at/training/other-courses/ngs-workshop-2014/"><em>Practical Course on Next Generation Sequencing for Population Genetics and Experimental Evolution</em></a>, 2014 Vet-Med Uni Vienna, A: Practicals about RNA-seq data analysis</li>
</ul>
</section>
<section id="as-participant" class="level2">
<h2 class="anchored" data-anchor-id="as-participant">As Participant</h2>
<ul>
<li><em>EMBO Training: Laboratory Leadership Course</em>, 2024, Budapest, H</li>
<li><em>An R Reproducibility Toolkit for the practical researcher</em>, 2022, online</li>
<li><em>8th International Course in Microbial Ecology ‚Äì Microbiome Metagenome Analysis</em>, 2017, CNR-ISE, Verbania, I</li>
<li><em>EMBO course: Bioinformatics and Comparative Genome Analyses</em>, 2011, Inst. Pasteur Paris, F</li>
<li><em>Computational Phyloinformatics Course</em>, 2008, Duke Univ., Durham, NC, USA</li>
<li><em>Workshop on Molecular Evolution</em>, 2006, Marine Biological Lab., Woods Hole, MA, USA</li>
</ul>
</section>
</section>
<section id="memberships" class="level1">
<h1>Memberships</h1>
<ul>
<li><a href="http://mabit.ttk.mta.hu/doc/en/home">Hungarian Society for Bioinformatics</a>: 2016 ‚Äì <strong>member of the directory board</strong>; 2006 ‚Äì member</li>
<li>Hungarian Academy of Sciences, Interdepartmental Scientific Committee on Bioinformatics: 2021 ‚Äì <strong>committee member</strong></li>
<li>Hungarian node of the <a href="https://elixir-europe.org/about-us/who-we-are/nodes/hungary">Elixir Europe</a>: 2023 ‚Äì <strong>lead researcher</strong></li>
<li><a href="https://scicomp.hu/?lang=en">Institute of Scientific Computing, Society</a> (SciComp): 2024 ‚Äì member</li>
<li>Public Body of the Hungarian Academy of Sciences: 2013 ‚Äì member (ID: 31344)</li>
</ul>
</section>
<section id="languages" class="level1">
<h1>Languages</h1>
<ul>
<li>Hungarian ‚Äì mother tongue</li>
<li>English ‚Äì C1 level</li>
<li>Italian ‚Äì A1 level</li>
</ul>
</section>
<section id="areas-of-interest" class="level1">
<h1>Areas of Interest</h1>
<ul>
<li>Bioinformatics</li>
<li>Evolutionary Biology</li>
<li>Antibiotic Resistance</li>
<li>Comparative genomics</li>
<li>Molecular phylogenetics</li>
<li>Metagenomics</li>
<li>Transcriptomics</li>
<li>Transcriptional regulation</li>
<li>Data analysis</li>
</ul>
</section>
<section id="professional-skills" class="level1">
<h1>Professional Skills</h1>
<ul>
<li>Developing and using different bioinformatic methods, software and pipelines.</li>
<li><em>R</em> programming language ‚Äì professional level</li>
<li><em>Linux Shell</em> scripting ‚Äì intermediate level</li>
</ul>
</section>
<section id="editing-reviewing-organizing-conferences" class="level1">
<h1>Editing, Reviewing, &amp; Organizing Conferences</h1>
<ul>
<li><a href="https://www.frontiersin.org/research-topics/62823/gene-evolution-and-mutation-in-evolutionary-systems-biology">Topic editor in Frontiers in Systems Biology</a></li>
<li>Reviewer in journals: <em>Brief. Bioinformatics</em> (3); <em>Evol. Biol.</em> (1); <em>Genome Biol. Evol.</em> (1); <em>J. Math. Chem.</em> (4); <em>Mol. Ecol.</em> (1); <em>Mol. Phylogenetics Evol.</em> (1); <em>Sci. Rep.</em> (1); <em>Opusc. Zool.</em> (1); <em>Plos One</em> (2)</li>
<li>Reviewing: PhD Theses (4), Grant proposals: Hungarian Scientific Research Fund Grant (4)</li>
<li>Conference organization as a member of the scientific organizing committee: <a href="https://www.congress-service.hu/rendezveny/met25">The Joint Conference of the Hungarian Physiological Society, the Hungarian Biophysical Society, the Hungarian Society for Microcirculation and Vascular Biology, and the Hungarian Bioinformatics Society</a>, Sep.&nbsp;2‚Äì5, 2025, Szeged, H</li>
</ul>
</section>
<section id="popularising-science" class="level1">
<h1>Popularising Science</h1>
<ul>
<li>Presenting at the Festival of ELTE: Global map of evolutionary dependencies between antibiotic resistance and virulence genes in <em>E. coli</em>. 2022, Budapest, H</li>
<li>Creating a website for The genomic epidemiology of SARS-CoV-2 in Hungary</li>
<li>Presenting at Capital of Science ‚ÄúTudom√°nyok F≈ëv√°rosa‚Äù: The genomic epidemiology of SARS-CoV-2 in Hungary, 2020, Budapest, H</li>
<li>Presenting at Night of Scientists ‚ÄúKutat√≥k √âjszak√°ja‚Äù, 2018 and 2020</li>
</ul>
</section>
<section id="scientific-websites" class="level1">
<h1>Scientific Websites</h1>
<ul>
<li>Supervising the creation of the <a href="https://tflink.net/">TFLink</a>, a transcription factor - target gene database.</li>
<li>Supervising the creation of the <a href="https://cran.r-project.org/web/packages/mulea/">mulea</a> and the <a href="https://bioconductor.org/packages/release/data/experiment/html/muleaData.html">muleaData</a> R packages for functional enrichment analysis and dataset.</li>
</ul>
</section>
<section id="publications" class="level1">
<h1>Publications</h1>
<ul>
<li><a href="https://scholar.google.com/citations?user=cYCKyr0AAAAJ"><img src="images/Google_Scholar_logo.svg.png" class="img-fluid" alt="Google Scholar" width="24"></a> <a href="https://m2.mtmt.hu/api/publication?cond=published;eq;true&amp;cond=core;eq;true&amp;cond=authors.mtid;eq;10020542&amp;sort=publishedYear,desc&amp;sort=firstAuthor,asc&amp;size=20&amp;fields=template,citations:0,citationCount&amp;cite_type=3&amp;&amp;page=1&amp;labelLang=eng"><img src="images/mtmt.jpg" class="img-fluid" width="24" alt="MTMT"></a></li>
<li>Nr. of published papers: <strong>27</strong> (D1: <strong>17</strong> + Q1: <strong>5</strong>)</li>
<li>Nr. of first, last or corresponding author publications: <strong>8</strong></li>
<li>Cumulative impact factor of first, last or corresponding author publications: <strong>47.22</strong></li>
<li>Cumulative impact factor of all publications (IF): <strong>218.56</strong></li>
<li>Independent citations of all publications (IC): <strong>600</strong></li>
<li>h-index: <strong>16</strong></li>
</ul>
<section id="preprints" class="level2">
<h2 class="anchored" data-anchor-id="preprints">Preprints</h2>
<ul>
<li>Balogh GM, Koncz B, Asztalos L, <strong>Ari E</strong>, Papp B, Szebeni GJ, G√©mes N, P√°l C &amp; Manczinger M (2025) APOBEC3-associated mutations generate immunogenic peptides in SARS-CoV-2. <em>Nature Communications</em>, under review.</li>
</ul>
</section>
<section id="published" class="level2">
<h2 class="anchored" data-anchor-id="published">Published</h2>
<ul>
<li><p>Daruka L, Czikkely MS, Szili P, Farkas Z, Balogh D, Maharramov E, Vu TH, Sipos L, Vincze BD, Gr√©zal G, Juh√°sz Sz, Dunai A, Daraba A, Sz√°mel M, S√°ri T, Stirling T, V√°s√°rhelyi BM, <strong>Ari E</strong>, Christodoulou C, Manczinger M, Enyedi MZs, Jaksa G, van Houte S, Pursey E, Papp CG, Szilovics Z, Pint√©r L, Haracska L, G√°cser A, Kintses B, Papp B &amp; P√°l Cs (2025) <a href="https://doi.org/10.1038/s41564-024-01891-8">ESKAPE pathogens rapidly develop resistance against antibiotics in development.</a> <em>Nature Microbiology</em>, accepted (preprint: <strong>bioRxiv</strong>, <a href="https://doi.org/10.1101/2023.07.23.550022">2023.07.23.550022</a>). <strong>D1, IF: 20.5, IC: 5</strong></p></li>
<li><p>Koncz M, Stirling T, Mehdi HH, M√©hi O, Eszenyi B, Asb√≥th A, Apjok G, T√≥th √Å, Orosz L, V√°s√°rhelyi BM, <strong>Ari E</strong>, Daruka L, Polg√°r TF, Schneider Gy, Zalokh SA, Sz√°mel M, Fekete G, Boh√°r B, Nagy Varga K, Visnyovszki √Å, Sz√©kely E, Licker MS, Izmendi O, Costache C, Gajic I, Lukovic B, Moln√°r Sz, Sz≈ëcs-Gazdi UO, Bozai Cs, Indreas M, Krist√≥f K, Van der Henst C, Breine A; P√°l C, Papp B &amp; Kintses B (2024) <a href="https://www.cell.com/cell/fulltext/S0092-8674(24)01027-4">Genomic surveillance as a scalable framework for precision phage therapy against antibiotic-resistant pathogens.</a> <em>Cell</em>, 187(21) 5901 - 5918.e28. <strong>D1, IF: 45.5, IC: 8</strong></p></li>
<li><p><span style="color: #00ab44;">Turek C, √ñlbei M, Stirling T, Fekete G, Tasn√°di E, Gul L, Boh√°r B, Papp B, Jurkowski W &amp; <strong>Ari E*</strong> (2024) <a href="https://doi.org/10.1186/s12859-024-05948-7">mulea - an R package for enrichment analysis using various ontologies and empirical false discovery rate.</a> <em>BMC Bioinformatics</em>, 25(1): 334. <strong>Q1, IF: 2.9, IC: 0 (*corresponding author)</strong></span></p></li>
<li><p><span style="color: #00ab44;">Gerber D, Szeifert B, Sz√©kely O, Egyed B, Gyuris B, Giblin JI, Horv√°th A, K√∂hler K, Kulcs√°r G, Kust√°r √Å, Major I, Moln√°r M, Palcsu L, Szever√©nyi V, F√°bi√°n S, Mende BG, Bond√°r M, <strong>Ari E*</strong>, Kiss V* &amp; Sz√©cs√©nyi-Nagy A* (2023) <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msad182/7240678">Interdisciplinary analyses of Bronze Age communities from Western Hungary reveal complex population histories.</a> <em>Molecular Biology and Evolution</em>, 40(9): msad182. <strong>D1, IF: 11, IC: 8 (*shared corresponding authorship)</strong></span></p></li>
<li><p>Bacsur P, Rutka M, Res√°l T, Sz√°nt√≥ K, J√≥j√°rt B, B√°lint A, <strong>Ari E</strong>, Walliyulah A, Kintses B, Feh√©r T, Asb√≥th A, Pigniczki D, Bor R, F√°bi√°n A, Mal√©th J, Szepes Z, Farkas K &amp; Moln√°r T (2023) <a href="https://journals.sagepub.com/doi/full/10.1177/17562848231174298">Effects of bowel cleansing on the composition of the gut microbiome in inflammatory bowel disease patients and healthy controls.</a> <em>Therapeutic Advances in Gastroenterology</em>, 16: 1-13. <strong>Q1, IF: 3.9, IC: 5</strong></p></li>
<li><p>Sturm √Å, Sask≈ëi √â, Hotzi B, Tarn√≥ci A, Barna J, Bodn√°r F, Sharma H, Kov√°cs T, <strong>Ari E</strong>, Weinhardt N, Kerepesi C, Perczel A, Ivics Z &amp; Vellai T (2023) <a href="https://doi.org/10.1038/s41467-023-40957-9">Downregulation of transposable elements extends lifespan in <em>Caenorhabditis elegans</em>.</a> <em>Nature Communications</em>, 14(1): 5278. <strong>D1, IF: 14.7, IC: 15</strong></p></li>
<li><p><span style="color: #00ab44;"><strong>Ari E</strong>, V√°s√°rhelyi BM, Kemenesi G, T√≥th GE, Zana B, Somogyi B, Lanszki Z, R√∂st G, Jakab F, Papp B &amp; Kintses B (2022) <a href="https://doi.org/10.1093/ve/veac069">A single early introduction governed viral diversity in the second wave of SARS-CoV-2 epidemic in Hungary.</a> <em>Virus Evolution</em>, 8(2): veac069. <strong>D1, IF: 5.3, IC: 4</strong></span></p></li>
<li><p>Csabai L, Fazekas D, Kadlecsik T, Szalay-Bek≈ë M, Boh√°r B, Madgwick M, M√≥dos D, √ñlbei M, Gul L, Sudhakar P, Kubisch J, Oyeyemi OJ, Liska O, <strong>Ari E</strong>, Hotzi B, Billes VA, Moln√°r E, F√∂ldv√°ri-Nagy L, Cs√°lyi K, Demeter A, P√°pai N, Koltai M, Varga M, Lenti K, Farkas IJ, T√ºrei D, Csermely P, Vellai T &amp; Korcsm√°ros T (2022) <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkab909/6389625">SignaLink3: A multi-layered resource to uncover tissue-specific signaling networks.</a> <em>Nucleic Acids Research</em>, 50(D1): 701-709. <strong>D1, IF: 14.9, IC: 26</strong></p></li>
<li><p><span style="color: #00ab44;">Liska O, Boh√°r B, Hidas H, Korcsm√°ros T, Papp B, Fazekas D &amp; <strong>Ari E*</strong> (2022) <a href="https://doi.org/10.1093/database/baac083">TFLink: An integrated gateway to access transcription factor - target gene interactions for multiple species.</a> <em>Database</em>, 2022, baac083. <strong>D1, IF: 5.8, IC: 73 ‚ÄúHigh impact paper‚Äù according to <a href="http://scientometrics.org">scientometrics.org</a>. (*corresponding author)</strong></span></p></li>
<li><p>B√°lint A, Farkas K, Kintses B, M√©hi O, V√°s√°rhelyi BM, <strong>Ari E</strong>, P√°l C, Mad√°csy T, Mal√©th J, Sz√°nt√≥ KJ, Nagy I, Rutka M, Bacsur P, Sz≈±cs D, Szepes Z, Nagy F, F√°bi√°n A, Bor R, Milassin √Å &amp; Moln√°r T (2020) <a href="https://www.mdpi.com/1424-8247/13/11/346/htm">Functional anatomical changes in ulcerative colitis patients determine their gut microbiota composition and consequently the possible treatment outcome.</a> <em>Pharmaceuticals</em>, 13(11): 346-362. <strong>D1, IF: 5.68, IC: 21</strong></p></li>
<li><p>Kintses B, Jangir PK, Fekete G, Sz√°mel M, M√©hi O, Spohn R, Daruka L, Martins A, Hosseinnia A, Gagarinova A, Kim S, Phanse S, Cs√∂rg√∂ B, Gy√∂rkei √Å, <strong>Ari E</strong>, L√°z√°r V, Farag√≥ A, Nagy I, Babu M, P√°l C &amp; Papp B (2019) <a href="https://www.nature.com/articles/s41467-019-13618-z">Chemical-genetic profiling reveals limited cross-resistance between antimicrobial peptides with different modes of action.</a> <em>Nature Communications</em>, -2208781740. <strong>D1, IF: 12.12, IC: 38</strong></p></li>
<li><p><span style="color: #00ab44;">Kintses B*, M√©hi O*, <strong>Ari E*</strong>, Sz√°mel M, Gy√∂rkei √Å, Jangir PK, Nagy I, P√°l F, Fekete G, Teng√∂lics R, Nyerges √Å, Lik√≥ I, B√°lint A, Moln√°r T, B√°lint B, V√°s√°rhelyi BM, Bustamante M, Papp B &amp; P√°l C (2019) <a href="https://www.nature.com/articles/s41564-018-0313-5">Phylogenetic barriers to horizontal transfer of antimicrobial peptide resistance genes in the human gut microbiota.</a> <em>Nature Microbiology</em>, 4(3): 447-458. <strong>D1, IF: 15.54, IC: 79 Craig MacLean recommended it in <a href="https://facultyopinions.com/prime/734649950#eval793557383">F1000Prime</a> as being of special significance in its field. It was selected as a part of the <a href="https://www.nature.com/collections/gjdhfgjiid">Nature Gut Microbiota core collection</a>. (*shared first authorship)</strong></span></p></li>
<li><p>Sudhakar P, Claire-Jacomin A, Hautefort I, Samavedam S, Fatemian K, <strong>Ari E</strong>, Gul L, Demeter A, Jones E, Korcsmaros T &amp; Nezis JP (2019) <a href="https://www.tandfonline.com/doi/full/10.1080/15548627.2019.1590519">Targeted interplay between bacterial pathogens and host autophagy.</a> <em>Autophagy</em>, 15(9): 1620-1633. <strong>D1, IF: 11.06, IC: 42</strong></p></li>
<li><p>Sun D, Ren X, <strong>Ari E</strong>, Korcsmaros T, Csermely P &amp; Wu L-Y (2019) <a href="https://academic.oup.com/bib/article-abstract/20/1/89/4079498?redirectedFrom=fulltext">Discovering cooperative biomarkers for heterogeneous complex disease diagnoses.</a> <em>Briefings in Bioinformatics</em>, 20(1): 89-101. <strong>D1, IF: 9.1, IC: 14</strong></p></li>
<li><p>Nyerges √Å, Cs√∂rg≈ë B, Draskovits G, Kintses B, Szili P, Ferenc Gy, R√©v√©sz T, <strong>Ari E</strong>, Nagy I, B√°lint B, V√°s√°rhelyi BM, Bihari P, Sz√°mel M, Balogh D, Papp H, Kalapis D, Papp B &amp; P√°l C (2018) <a href="https://www.pnas.org/content/115/25/E5726">Directed evolution of multiple genomic loci allows the prediction of antibiotic resistance.</a> <em>Proceedings of the National Academy of Sciences of the United States of America</em>, 115(25): E5726-E5735. <strong>D1, IF: 9.58, IC: 76</strong></p></li>
<li><p>Fodor E, Sigmond T, <strong>Ari E</strong>, Lengyel K, Tak√°cs-Vellai K, Varga M &amp; Vellai T (2017) <a href="https://www.sciencedirect.com/science/article/pii/S0076687916303949">Methods to study autophagy in zebrafish.</a> <em>Methods in Enzymology</em>, 588: 467-96. <strong>Q1, IF: 1.48, IC: 19</strong></p></li>
<li><p>M√©tris A, Sudhakar P, Fazekas D, Demeter A, <strong>Ari E</strong>, √ñlbei M, Branchu P, Kingsley RA, Baranyi J &amp; Korcsm√°ros T (2017) <a href="https://www.nature.com/articles/s41540-017-0034-z">SalmoNet, an integrated network of ten <em>Salmonella enterica</em> strains reveals common and distinct pathways to host adaptation.</a> <em>Npj Systems Biology and Applications</em>, 3(1): s41540-17-34-NaN-17. <strong>Q1, IF: 3.81, IC: 28</strong></p></li>
<li><p>Ricotta C, <strong>Ari E</strong>, Bonanomi G, Giannino F, Heathfield D, Mazzoleni S &amp; Podani J (2017) <a href="https://akjournals.com/view/journals/168/18/1/article-p37.xml">Spatial analysis of phylogenetic community structure: New version of a classical method.</a> <em>Community Ecology</em>, 18(1): 37-46. <strong>Q2, IF: 0.98, IC: 2</strong></p></li>
<li><p>√Åcs √â, <strong>Ari E</strong>, Duleba M, Dressler M, Genkal SI, Jak√≥ √â, Rimet F, Ector L &amp; Kiss KT (2016) <a href="https://fottea.czechphycology.cz/artkey/fot-201601-0006_pantocsekiella_a_new_centric_diatom_genus_based_on_morphological_and_genetic_studies.php"><em>Pantocsekiella</em>, a new centric diatom genus based on morphological and genetic studies.</a> <em>Fottea</em>, 16(1): 56-78. <strong>Q2, IF: 0.6, IC: 52</strong></p></li>
<li><p><strong>Ari E*</strong> &amp; Jak√≥ √â (2016) <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/cem.2781">Classification of tRNA isoacceptor sequences by using graph-based molecular descriptors.</a> <em>Journal of Chemometrics</em>, 30(1): 182-187. <strong>Q2, IF: 2.03, IC: 0 (*corresponding author)</strong></p></li>
<li><p>M√≥dos D, Brooks J, Fazekas D, <strong>Ari E</strong>, Vellai T, Csermely P, Korcsm√°ros T &amp; Lenti K (2016) <a href="https://www.nature.com/articles/srep38588">Identification of critical paralog groups with indispensable roles in the regulation of signaling flow.</a> <em>Scientific Reports</em>, 6: 38588. <strong>D1, IF: 4.62, IC: 6</strong></p></li>
<li><p>F√∂ldv√°ri-Nagy L, <strong>Ari E</strong>, Csermely P, Korcsm√°ros T &amp; Vellai T (2014) <a href="https://www.nature.com/articles/srep05829">Starvation-response may not involve Atg1-dependent autophagy induction in non-unikont parasites.</a> <em>Scientific Reports</em>, 4: 5829. <strong>D1, IF: 5.85, IC: 16</strong></p></li>
<li><p><span style="color: #00ab44;"><strong>Ari E*</strong>, Ittz√©s P, Podani J, Le Thi QC &amp; Jak√≥ √â (2012) <a href="https://www.sciencedirect.com/science/article/abs/pii/S1055790312000279">Comparison of Boolean analysis and standard phylogenetic methods using artificially evolved and natural mt-tRNA sequences from great apes.</a> <em>Molecular Phylogenetics and Evolution</em>, 63(1): 193-202. <strong>D1, IF: 4.42, IC: 5 (*corresponding author)</strong></span></p></li>
<li><p>Boros G, Cech G, <strong>Ari E</strong> &amp; D√≥zsa-Farkas K (2010) <a href="https://www.tandfonline.com/doi/abs/10.1080/09397140.2010.10638454">Extension of employing ITS region in the investigation of Hungarian <em>Fridericia</em> species (Oligochaeta: Enchytraeidae).</a> <em>Zoology in the Middle East</em>, 51(sup2): 23-30. <strong>Q3, IF: 0.3, IC: 2</strong></p></li>
<li><p>Tihanyi B, Vellai T, Reg≈ës √Å, <strong>Ari E</strong>, M√ºller F &amp; Tak√°cs-Vellai K (2010) <a href="https://bmcdevbiol.biomedcentral.com/articles/10.1186/1471-213X-10-78">The <em>C. elegans</em> Hox gene <em>ceh-13</em> regulates cell migration and fusion in a non-colinear way. Implications for the early evolution of Hoxclusters.</a> <em>BMC Developmental Biology</em>, 10(1): 78-92. <strong>Q1, IF: 2.78, IC: 26</strong></p></li>
<li><p>Jak√≥ √â, <strong>Ari E</strong>, Ittz√©s P, Horv√°th A &amp; Podani J (2009) <a href="https://www.sciencedirect.com/science/article/abs/pii/S1055790309001584">BOOL-AN: A method for comparative sequence analysis and phylogenetic reconstruction.</a> <em>Molecular Phylogenetics and Evolution</em>, 52(3): 887-897. <strong>D1, IF: 3.88, IC: 30</strong></p></li>
<li><p><strong>Ari E</strong>, Ittz√©s P, Podani J &amp; Jak√≥ √â (2008) <a href="https://www.researchgate.net/publication/216330267_Phylogenetic_Tree_Reconstruction_with_a_New_Discrete_Mathematical_Method">Phylogenetic tree reconstruction with a new discrete mathematical method</a> <em>Kitaibelia</em>, Kitaibelia 13: 209-211. <strong>Q2, IF: 0.23, IC: 0 (in Hungarian)</strong></p></li>
</ul>
</section>
</section>
<section id="oral-presentations-in-conferences" class="level1">
<h1>Oral Presentations in Conferences</h1>
<ul>
<li>Nr. of international conference talks: <strong>9</strong></li>
<li>Nr. of national conference talks: <strong>6</strong></li>
<li>As an invited speaker*: <strong>3</strong></li>
</ul>
<!-- Insert a blank line here -->
<ul>
<li>Ari E, <em>et al.</em> (2024) mulea - an R package for enrichment analysis using multiple ontologies and empirical false discovery rate. <a href="https://www.mabit.org.hu/doc/hu/society/conferences/2024/bioinf_2024/1"><em>Bioinformatika, a Magyar Tudom√°ny √únnep√©n</em></a>, 8 Nov, Budapest, H</li>
<li>Ari E, <em>et al.</em> (2024) mulea - an R package for enrichment analysis using multiple ontologies and empirical false discovery rate. <a href="https://indico.wigner.hu/event/1583/overview"><em>SciComp24 Conference</em></a>, 17-19 Oct, Szeged, H</li>
<li>Ari E, <em>et al.</em> (2023) Global map of evolutionary dependencies between antibiotic resistance and virulence genes in <em>E. coli</em>. <a href="https://www.hcemm.eu/news/2023/11/3rd-hpps-phd-postdoc-symposium-keszthely/"><em>3rd HCEMM PhD-POSTDOC Symposium</em></a>, 8-9 Nov, Keszthely, H</li>
<li>Ari E, <em>et al.</em> (2022) Global map of evolutionary dependencies between antibiotic resistance and virulence genes in <em>E. coli</em>. <a href="https://konferencia.unideb.hu/en/BDG2022"><em>1st Bioinformatics and Data Science in Genomic Studies (BDG2022)</em></a> online conference, 25 Nov, University of Debrecen, H</li>
<li>Ari E, <em>et al.</em> (2022) Global map of evolutionary dependencies between antibiotic resistance and virulence genes in <em>E. coli</em>. <em>Bioinformatics, Celebrating the Hungarian Science</em>, 11 Nov, HAS, Research Centre for Natural Sciences, Budapest, H</li>
<li>Ari E*, Kintses B (2020) Methods to investigate the microbiome and understand the results. <em>Meeting of Hungarian Society for Gastroenterology, Section Colon</em>, 6-7 Mar, Visegr√°d, H</li>
<li>Ari E, <em>et al.</em> (2019) Phylogenetic barriers to horizontal transfer of antimicrobial peptide resistance genes in the human gut microbiota. <em>EvolBiol Day</em>, 17 Apr, Szeged, H</li>
<li>Ari E*, <em>et al.</em> (2018) How well do the antimicrobial peptide resistance genes spread? <em>Bioinformatics, Celebrating the Hungarian Science</em>, 16 Nov, HAS, Research Centre for Natural Sciences, Budapest, H</li>
<li>Ari E: (2017) MulEA ‚Äì A tool for multi-enrichment analysis. <em>2nd Interdisciplinary Signaling Workshop</em>, 17-21 Jul, Visegr√°d, H</li>
<li>Ari E*: (2012) Molecular phylogenetic reconstructions with a discrete mathematical method, the Boolean analysis. <em>International Congress, Hungarian Society for Microbiology</em>, 24-26 Oct, Keszthely, H</li>
<li>Ari E &amp; Jak√≥ √â (2012) Comparison of Boolean analysis and standard phylogenetic methods using artificially evolved and natural mt-tRNA sequences from great apes. <em>9th Joint Conference on Mathematics and Computer Science</em>, 9-12 Feb, Si√≥fok, H</li>
<li>Ari E, <em>et al.</em> (2007) Boolean analysis: A new discrete mathematical method for phylogenetic reconstruction. <em>From Molecular Informatics to Bioinformatics ‚Äì International Symposium</em>, 19-21 Apr, Collegium Budapest, H</li>
<li>Ari E &amp; Jak√≥ √â (2007) Testing a new discrete mathematical method for phylogenetic reconstruction. <em>Evolution 2007 Conference</em>, 16-20 Jun, Christchurch, NZ</li>
<li>Ari E &amp; Jak√≥ √â (2007) Testing a new discrete mathematical method for phylogenetic reconstruction. <em>The Dumont D‚ÄôUrville Workshop on Applied Evolutionary Bioinformatics</em>, 24-27 Jun, Kaikoura, NZ</li>
<li>Ari E, <em>et al.</em> (2007) Phylogenetic tree reconstruction with a new discrete mathematical method. <em>Molecular taxonomic, phylogenetic and phylogeographic researches in Hungary, Meeting in memory of S√°muel Di√≥szegi</em>, 17 Nov, Debrecen, H</li>
<li>Ari E (2006) Reconstructing the phylogenetic tree of great apes by using a new discrete mathematical method. <em>12th Annual European Meeting of PhD students in Evolutionary Biology</em>, 4-9 Sep, St.&nbsp;Andrews, UK</li>
</ul>
</section>
<section id="other-invited-talks" class="level1">
<h1>Other Invited Talks</h1>
<ul>
<li>Ari E* (2017) Investigating the antimicrobial peptide resistome in the human gut microbiome: a metagenomic approach. University of Graz, Institute of Zoology, 15 Nov, Graz, A</li>
<li>Ari E*: (2015) Rapid evolution of phenotypic plasticity during experimental evolution of <em>Drosophila</em>. The Genome Analysis Centre, 23 Feb, Norwich, UK</li>
</ul>
</section>
<section id="poster-presentations-in-conferences" class="level1">
<h1>Poster Presentations in Conferences</h1>
<ul>
<li>Nr. of international conference posters: <strong>9</strong></li>
</ul>
<!-- Insert a blank line here -->
<ul>
<li>Ari E, <em>et al.</em> (2023) <a href="https://indico.ictp.it/event/10210/material/4/0.pdf">Global map of evolutionary dependencies between antibiotic resistance and virulence genes in <em>E. coli</em></a>, <a href="https://meetings.embo.org/event/23-amr-plasmids"><em>EMBO Workshop: Plasmids as vehicles of AMR spread</em></a>, 12-18 Sep, (Trieste, I) online</li>
<li>Ari E, <em>et al.</em> (2023) Global map of evolutionary dependencies between antibiotic resistance and virulence genes in <em>E. coli</em>, <a href="https://www.embl.org/about/info/course-and-conference-office/events/pev23-01/"><em>EMBO Workshop: Predicting evolution</em></a>, 11-14 Jul, Heidelberg, D</li>
<li>Ari E, <em>et al.</em> (2022) <a href="https://www.lamg.info/_files/ugd/2e355c_d62492daad5840479e27865c3233bfba.pdf">Global map of evolutionary dependencies between antibiotic resistance and virulence genes in <em>E. coli</em></a>. <a href="https://www.lamg.info/"><em>Lake Arrowhead Microbial Genomics Conference</em></a>, 11-15 Sep, Lake Arrowhead, CA, USA</li>
<li>Rutka M, Sz√°nt√≥ K, Bacsur P, Res√°l T, J√≥j√°rt B, B√°lint A, <strong>Ari E</strong>, Kintses B, Feh√©r T, Asb√≥th A, Pigniczki D, Bor R, F√°bi√°n A, Farkas K, Mal√©th J, Szepes Z &amp; Moln√°r T (2022) <a href="https://academic.oup.com/ecco-jcc/article/16/Supplement_1/i609/6513059">P713 Gut microbiota alterations after bowel preparation amongst inflammatory bowel disease patients</a>. <em>Journal of Crohn‚Äôs and Colitis</em>, 16 (Supplement_1): i609. <strong>Q1, IF: 9.07</strong></li>
<li>Ari E, <em>et al.</em> (2019) <a href="https://www.researchgate.net/publication/338231106_Phylogenetic_barriers_to_horizontal_transfer_of_AMP_resistance_genes">Phylogenetic barriers to horizontal transfer of antimicrobial peptide resistance genes in the human gut microbiota</a>. <em>Gordon Research Conference on Molecular Mechanisms in Evolution</em>, 9-14 Jun, Easton, MA, USA</li>
<li>Ari E, <em>et al.</em> (2019) <a href="https://www.researchgate.net/publication/338231106_Phylogenetic_barriers_to_horizontal_transfer_of_AMP_resistance_genes">Phylogenetic barriers to horizontal transfer of antimicrobial peptide resistance genes in the human gut microbiota</a>. <em>Gordon Research Seminar on Molecular Mechanisms in Evolution</em>, 8-9 Jun, Easton, MA, USA</li>
<li>Ari E &amp; Jak√≥ √â (2015) Graph-based generalized Boolean descriptors for classification of biological macromolecules. <em>Conferentia Chemometrica</em>, 13-16 Sep, Budapest, H</li>
<li>Ari E <em>et al.</em> (2015) Rapid evolution of phenotypic plasticity during experimental evolution of Drosophila. <em>Annual Meeting of the Society for Molecular Biology and Evolution</em>, 12-16 Jul, Vienna, A</li>
<li>Ari E, <em>et al.</em> (2006) Testing a new discrete mathematical method for reconstructing the great apes phylogeny based on mitochondrial tRNAs. <em>Evolution of Biomolecular Structure</em>, 25-27 May, Vienna, A</li>
</ul>


</section>


<div id="quarto-appendix" class="default"><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><h2 class="anchored quarto-appendix-heading">Footnotes</h2>

<ol>
<li id="fn1"><p>Formal names: Research Network of the Hungarian Academy of Science (HAS); E√∂tv√∂s Lor√°nd Research Network (ELKH)<a href="#fnref1" class="footnote-back" role="doc-backlink">‚Ü©Ô∏é</a></p></li>
<li id="fn2"><p>Formal and actual name: University of Veterinary Medicine<a href="#fnref2" class="footnote-back" role="doc-backlink">‚Ü©Ô∏é</a></p></li>
</ol>
</section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "Óßã";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
          // target, if specified
          link.setAttribute("target", "_blank");
          if (link.getAttribute("rel") === null) {
            link.setAttribute("rel", "noopener");
          }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>